readme.md

```diff
@@ -170,7 +172,20 @@ You can write your own client simply by implementing these interfaces:
 - Config must implement `StrictPhp\HttpClients\Contracts\ConfigContract`
 - Factory for client implement `StrictPhp\HttpClients\Contracts\ClientFactoryContract`
 
-Below is an example of a implementation:
+Below is an example of an implementation:
+
+## Testing
+
+You can save file with extension *.shttp by [SaveResponse](src/Responses/SaveResponse.php) or use our PSR16 implementation [CachePsr16Service](src/Services/CachePsr16Service.php). This file you can to use in tests by [CustomResponseClient](src/Clients/CustomResponse/CustomResponseClient.php).
```

Comment:
proč je to shttp? on to není formát pro použítí v PHPStormu? (tam je to jenom .http)

Reply:
http se generuje také, shttp = serialized http